tekton/release-pipeline.yaml

```diff
@@ -66,7 +76,16 @@ spec:
     - name: precheck
       runAfter: [git-clone]
       taskRef:
-        name: prerelease-checks
+        resolver: git
+        params:
+        - name: repo
+          value: pipeline
```

Comment:
The `prerelease-check` comes from the plumbing repo. It is deployed together with the pipeline to the cluster when the pipeline definition is installed, so maybe we don't need to change this for now. 
If you wanted to use remote resolution here, this is the URL https://github.com/tektoncd/plumbing/blob/main/tekton/resources/release/base/prerelease_checks.yaml